exercism-modern.el

```diff
@@ -232,11 +263,20 @@ Pass prefix BUFFER-PREFIX-ARG to prompt for a buffer instead."
                                            (is-unlocked (not (eq :json-false (alist-get 'is_unlocked exercise))))
                                            (is-recommended (not (eq :json-false (alist-get 'is_recommended exercise))))
                                            (text-face (if is-unlocked 'default 'shadow))
-                                           (difficulty-svg (exercism-modern--load-from-package-root (concat "icons/" difficulty ".svg"))))
+                                           (difficulty-svg (symbol-value (intern (concat "exercism-modern-" difficulty "-icon")))))
                                       (list slug
-                                            (vector (concat
-                                                     (when is-recommended
-                                                       (propertize "  " 'face 'exercism-modern-warning))
+                                            (vector (if is-recommended
```

Comment:
when is-recommended (didn't I already commit that?)

Reply:
But there's an `else` part for an empty string?